### ISSUE TITLE
The accuracy of the best result

### ISSUE THREAD

**Issue by OlivierX** [yl-1993/Matrix-Capsules-EM-PyTorch#5]
The paper reaches a much lower Err-rate(about 1.5%) for smallNORB, but it seems that no one here can achieve the level of state-of-art.  i wonder why : )

**Comment by ZHAOZHIHAO** [comment]
I'm curious about this question too. I think this may happen even if this matrix capsule network is totally correctly implemented, because some people say it's hard to train. See https://www.reddit.com/r/MachineLearning/comments/a83fpx/r_what_is_the_status_about_the_capsule_networks/

**Comment by yl-1993** [comment]
Thanks for the discussion. There are also other issues or blogs concerning this question:
- https://github.com/naturomics/CapsLayer/issues/30
- https://yhyu13.github.io/2017/12/17/Matrix-Capsule-With-EM-Routing-Reproduce-Report/

Reproduce the CNN baseline maybe a good start point to explore this question. (to make sure other factors such as data augmentation to be the same with the paper)

Feel free to discuss if you make any progress.
